WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -34,6 +40,19 @@ public CoinJoinManager(WalletManager walletManager, RoundStateUpdater roundStatu
 	private TimeSpan AutoCoinJoinDelayAfterWalletLoaded { get; } = TimeSpan.FromMinutes(Random.Shared.Next(5, 16));
 	public bool IsUserInSendWorkflow { get; set; }
 
+	private List<(string WalletName, CoinJoinCommand Action)> WalletActions { get; } = new();
+	public void Start(string walletName)
```

Comment:
These methods would be nicer if we could pass in the Wallet itself, instead of a string. (Ambiguity)

Reply:
done